### ISSUE TITLE
Clarify "Parent has open child tasks" error

### ISSUE THREAD

**Issue by hschallhorn** [department-of-veterans-affairs/caseflow#12515]
[Relevant slack chat](https://dsva.slack.com/archives/C6E41RE92/p1571335729037500). When VLJs attempt to assign a task to someone in certain scenarios, they receive a cryptic error of "Parent has open child tasks". Instead, we should be presenting the user with an error that provides context for what they're attempting to complete with their workflow. e.g. "Drafting decisions for this case has already been assigned to an attorney (<CSS_ID>). If you are attempting to assign this case to a different attorney, use the "Assign to attorney" menu item on the case details screen."

### Notes
Check style guide for content norms on this wording. Check with design channel about wording with visual of current error and context before starting coding.

### Out of scope
- [ ] Do followup research on contextualizing these messages.

### AC
- [ ] Figure out the root cause of these errors (does not seem to be when judges attempt to assign an attorney task when it is already assigned)
- [ ] Error message for duplicate child tasks conforms to caseflow error style

**Comment by jimruggiero** [comment]
_[What is this graph?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```
1 | 
2 | ||||||||||
3 | ||||||||
5 | 
8 | 
```
Uncertainty around isolating the error. Error appears to be infrequent and low priority, since it's recoverable.

**Comment by laurjpeterson** [comment]
@carodew FYI, this content ticket exists!
